recipes/py-videodev2/recipe.yaml

```diff
@@ -0,0 +1,43 @@
+context:
+  version: 0.0.4
+
+package:
+  name: py-videodev2
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/v/videodev2/videodev2-${{ version }}.tar.gz
+    sha256: c34ba70491d148c23a08cbacd8efabeb413cff5baa943a7548ac4abd1eb19e2a
+
+build:
+  noarch: python
+  number: 0
+  script: ${{ PYTHON }} -m pip install .
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - if: linux
+      then:
+        - __linux
```

Comment:
I think it make sense to have the dependency unconditionally. You do not want to be able to build a non-functional package on macos and windows.